Badge1.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Badge1.css">
+
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>Badge on icon</h1>
+    <div class="badge-container">
+        <i class="fas fa-bell fa-2x"></i>
+        <span class="badge badge-color-1"></span>
+    </div>
+
+    <div class="badge-container">
+        <i class="fas fa-bell fa-2x"></i>
+        <span class="badge badge-color-2 badge-2">5</span>
+    </div>
+
+    <h1>Badge on Avatar</h1>
+    <div class="badge-container">
+        <img src="/images/img_avatar.png"  class="avatarImg"alt="">
+        <span class="badge badge-color-1 badge-position-bottom"></span>
+    </div>
+
+    <div class="badge-container">
+        <img src="/images/img_avatar.png"  class="avatarImg"alt="">
+        <span class="badge badge-color-3 badge-position-bottom"></span>
+    </div>
+
+    <div class="badge-container">
+        <img src="/images/img_avatar.png"  class="avatarImg"alt="">
+        <span class="badge badge-color-4 badge-position-bottom"></span>
+    </div>
+
+
+    <script src="https://kit.fontawesome.com/228add9a8b.js" crossorigin="anonymous"></script>
```

Comment:
font awesome kit should be in <head> tag. Because it is only used for fonts and doesn't have any functionality to do on the website. Correct me if wrong.